README.md

```diff
@@ -1,12 +1,25 @@
 # LTI 2.1 Reference Tool Provider
 An LTI tool provider intended to be used as a reference for implementing the IMS LTI 2.1 specification. Section numbers in the comments (i.e. “6.1.2”) refer to sections of the IMS LTI 2.1 specification.
 
-## Setup
+## Running locally
+### Setup
 1. `bundle install`
 2. `bundle exec rake db:create`
 3. `bundle exec rake db:migrate`
 4. `bundle exec rackup`
 
-## Running Tests
+### Running tests
 1. `bundle exec rake db:migrate RACK_ENV=test`
```

Comment:
the dockerfile and docker-compose.yml files are set to run in  production. So there isn't any test or development dependencies installed